provisioning_templates/PXELinux/autoyast_default_pxelinux.erb

```diff
@@ -5,8 +5,14 @@ oses:
 - SLES
 - OpenSUSE
 %>
+<%
+  http_proxy_enabled = @host.params['http-proxy'] ? true : false
```

Comment:
@mmol yeah, it looks a bit messy. I’ll figure it out.
@ekohl good question, will test on tuesday.
@mmoll The nice thing about this solution it uses the proxy only during installation. I used about the same code in the provisioning template, setting up `<proxy>…</proxy>` and that didn’t do anything. But after installation I can use puppet to set up the system like I want.